### DISCUSSION TITLE
How to start to understand and learn this technology?

### DISCUSSION THREAD

**Discussion by XiaoYaoheihei** [apache/pulsar#19807] [Q&A] [answered]
Hi,team,
I'm a new member of the community, I want to start to understand and learn this technology, is there any good advice or route?
Thank you!

**Comment by jak78** [comment, marked as answer]
Hi @XiaoYaoheihei, and welcome to the community!
I would recommend enrolling in courses from [StreamNative Academy](https://www.academy.streamnative.io/tracks) and you can start with this one: [Introduction to Apache Pulsar](https://www.academy.streamnative.io/courses/course-v1:streamnative+Beg-001+2022/about)
You can also watch videos from their [YouTube channel](https://www.youtube.com/playlist?list=PLqRma1oIkcWjkpm1-0ox-BFJdVBsgZwPU).
You can also read the [Apache Pulsar in Action book](https://streamnative.io/ebooks/get-your-free-copy-of-mannings-apache-pulsar-in-action).
If you want to 'feel' Pulsar by trying it out, you can follow [one of these Getting Started guides](https://pulsar.apache.org/docs/2.11.x/getting-started-home/).
Also, @visortelle has built an amazing list of curated Pulsar resources: https://github.com/tealtools/awesome-apache-pulsar
Don't hesitate to reach out if you have any questions during your journey through Pulsar. I'd be happy to help 😊

**Reply by XiaoYaoheihei** [reply]
Thank you !!!
